extension/scripts/lib/tests/scroll.js

```diff
@@ -0,0 +1,32 @@
+// This test runs on the client side.
```

Comment:
Can you add some more detail, e.g. The purpose of the test and assumptions you've made whilst writing it.

e.g. This tests for scroll events by keeping track of scroll listeners.

The one for thirdParty:

```

/**
 * This runs in the devtools page
 *
 * Tracks requests up until the page load event is received and then it returns.
 * Promise rejects if any third party requests are received;
 *
 * NOTE: This does not track items received via the cache API aka, from a service worker.
 */
```